pyQuARC/schemas/rule_mapping.json

```diff
@@ -2565,6 +2565,35 @@
             ]
         },
         "severity": "info",
+        "check_id": "one_item_presence_check"    
+    },
+    "standard_product_check": {
+        "rule_name": "Standard Product Check",
+        "fields_to_apply": {
+            "echo-c": [
+                {
+                    "fields": [
+                        "Collection/StandardProduct"
+                    ]
+                }
+             ],
+            "dif10": [    
+                {
+                    "fields": [
+                        "DIF/Metadata/Value",
+                        "DIF/Extended_Metadata/Metadata/Value"
```

Comment:
@jenny-m-wood are you okay with us merging these branches since it's currently working with the schema now? it won't be going into master before we check the new schema